index.html

```diff
@@ -4,35 +4,53 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>MyStayl</title>
   <link rel="stylesheet" href="my.css">
+  <script src="app.js"></script>
+  <title>Document</title>
 </head>
- <style>
-Ul {
-  list-style-type: none;
-  margin: 0;
-  padding: 0;
-} 
-</style>
-</head>
-<body> 
-  <div class="qqqq">
-    <a class="wwwww" href="default.asp">Home</a>
-    <a href="news.asp">News</a>
-    <a href="contact.asp">Contact</a>
-    <a href="about.asp">About</a>
-  </div>
-  <div class="eeee">
-    <a class="rrrr" href="default.asp">Home</a>
-    <a href="news.asp">News</a>
-    <a href="contact.asp">Contact</a>
-    <a class="tttt" href="about.asp">About</a>
+<body>
+  <div class="wrapper">
+    <header class="header">
+      <div class='container'>
+        <div class="headerbody">
+          <a href="#" class="headerlogo">
+            <img src="logo.jpg" alt="">
+          </a>
+          <div class="headerburger" onclick="onoff()">
+            <span></span>

```

Comment:
You don't need to use this span. Please remove this.

Reply:
remove <span>